<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="5" hostname="A2ML30049" name="io.mosip.registration.main.PacketCreation" tests="6" failures="0" timestamp="11 Sep 2019 06:03:31 GMT" time="31.255" errors="1">
  <testcase name="validatePacketCreation" time="0.002" classname="io.mosip.registration.main.PacketCreation">
    <skipped/>
  </testcase> <!-- validatePacketCreation -->
  <testcase name="validatePacketCreation" time="0.002" classname="io.mosip.registration.main.PacketCreation">
    <skipped/>
  </testcase> <!-- validatePacketCreation -->
  <testcase name="validatePacketCreation" time="0.005" classname="io.mosip.registration.main.PacketCreation">
    <skipped/>
  </testcase> <!-- validatePacketCreation -->
  <testcase name="dataSetUp" time="31.246" classname="io.mosip.registration.main.PacketCreation">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at io.mosip.registration.main.BaseConfiguration.baseSetUp(BaseConfiguration.java:108)
at io.mosip.registration.main.PacketCreation.dataSetUp(PacketCreation.java:98)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at io.mosip.registration.main.RegClient.createPacket(RegClient.java:25)
at io.mosip.registration.main.RegClient.main(RegClient.java:31)
]]>
    </error>
  </testcase> <!-- dataSetUp -->
  <testcase name="validatePacketCreation" time="0.000" classname="io.mosip.registration.main.PacketCreation">
    <skipped/>
  </testcase> <!-- validatePacketCreation -->
  <testcase name="validatePacketCreation" time="0.000" classname="io.mosip.registration.main.PacketCreation">
    <skipped/>
  </testcase> <!-- validatePacketCreation -->
</testsuite> <!-- io.mosip.registration.main.PacketCreation -->
